target/linux/mvebu/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 BOARD:=mvebu
```

Comment:
Impossible to comment on commit description, so adding it here.

> This has two benefits:
> 1. Debloats kernel size for Armada 370/XP, especially Linksys WRT1900AC (Mamba)

This is true only partially as after the split there is now wrt1900ac-v1 (3072k) in cortexa9.mk and wrt32x (3072k) in cortexa9_neon.mk, so this debloat reasoning doesn't make much sense.

If there's a problem with kernel size and it can't be fixed by other means, then the split should be into `tiny` subtarget with both tiny devices in there.